src/config.ts

```diff
@@ -13,7 +13,7 @@ const config: ApplicationConfiguration = {
         cacheServer: process.env.CACHE_SERVER as string
     },
     applicationNamespace: "restricted-word-web",
-    baseUrl: process.env.CHS_URL ?? "https://chs.local"
+    baseUrl: process.env.CHS_URL ?? "http://chs.local"
```

Comment:
Wondering about this change. Why was it done?

Possibly this is PA's change so would be good to get that merge of 'master' into node 18 branch to make reviewing this easier.

Reply:
I've updated the base branch with master and this has now disappeared.